scripts/genproto.sh

```diff
@@ -114,10 +114,10 @@ schwag -input=Documentation/dev-guide/apispec/swagger/rpc.swagger.json
 # install protodoc
 # go get -v -u github.com/coreos/protodoc
 #
-# by default, do not run this option.
-# only run when './scripts/genproto.sh -g'
+# run './scripts/genproto.sh skip-protodoc'
```

Comment:
will `--skip-protodoc` matches the convention of command flag?

Reply:
@fanminshi Yeah just changed it to `--skip-protodoc`.